extensions/docs/msal-node-extensions.md

```diff
@@ -11,17 +11,49 @@ Supported platforms are Windows, Mac and Linux:
 - Linux - LibSecret is used for storing to "Secret Service" through npm keytar.
 
 ## Code
+
 ### Creating the persistence layer
-Creating the persistence will differ based on what platform you are targeting.
 
+API for creating the persistence layer will differ based on what platform you are targeting.
+
+Alternatively, you can use PersistenceCreator.createPersistence as it's a generic wrapper and selects the appropriate persistence based on the platform . 
+
+```js
+const { protectedResources, msalConfig, persistenceConfiguration } = require("./authConfig");
+const {
+  DataProtectionScope,
+  PersistenceCreator,
+  PersistenceCachePlugin,
+} = require("@azure/msal-node-extensions");
+
+let authProvider;
+authProvider = new AuthProvider(msalConfig);
+...
+
+PersistenceCreator
+        .createPersistence(persistenceConfiguration)
+        .then((persistence) => {
+            msalConfig.cache.cachePlugin = new PersistenceCachePlugin(persistence);
+
+            authProvider = new AuthProvider(msalConfig); //pass the msalConfig to AuthProvider constructor
+        });
```

Comment:
```suggestion
const { PublicClientApplication } = require("@azure/msal-node");

const {
  DataProtectionScope,
  PersistenceCreator,
  PersistenceCachePlugin,
} = require("@azure/msal-node-extensions");

PersistenceCreator
        .createPersistence({
                cachePath: "path/to/cache/file.json",
                dataProtectionScope: DataProtectionScope.CurrentUser,
                serviceName: "test-msal-electron-service",
                accountName: "test-msal-electron-account",
                usePlaintextFileOnLinux: false,
          }).then((persistence) => {
          
            // use the persistence object to initalize an 
            // MSAL PublicClientApplication with cachePlugin
            const pca = new PublicClientApplication({
                    auth: {
                        clientId: "CLIENT_ID_HERE",
                    },
                    cache: {
                        cachePlugin: new PersistenceCachePlugin(persistence);
                    },
                });
        });
```

Reply:
wanted to make this snippet more self contained, but feel free to edit! (I think we won't need the last snippet down below (authConfig.js) after this)